### PR TITLE
Vb/feat/230/custom scalars part 1 - standard scalars and example

### PR DESCRIPTION
**Pull request by vadimbasko**

### CODE REVIEW

File: example-app/src/core/transform/model/deserialize.ts [hunk 1]
```diff
@@ -2,10 +2,10 @@ import { NullableObjectOrList } from "../types";
...
 
 /**
- * Used to process fetched data (for example, data received from `useQuery`).
- * Apollo Client deserializes built-in scalars, but not custom scalars, therefore data will contain
- * custom scalar fields in serialized format. These fields will be deserialized using the functions
- * in {@link customScalarTransformers}.
+ * Adjust backend data for frontend usage.
+ *
+ * In order to change how custom scalars are deserialized,
+ * make changes to {@link customScalarTransformers}
```
Comment (vyacheslav-pushkin) [inline example-app/src/core/transform/model/deserialize.ts:8]: Looks like accidental rewrite with older doc version

File: example-app/src/core/transform/model/serialize.ts [hunk 2]
```diff
@@ -3,10 +3,9 @@ import { NullableObjectOrList } from "../types";
 import { transform } from "../transform";
 
 /**
- * Used to process data that is intended to be sent to backend (for example, a mutation input).
- * Apollo Client serializes built-in scalars, but not custom scalars. 
- * This function will serialize custom scalar fields using the functions
- * in {@link customScalarTransformers}.
+ * Serialize form data before sending to server - additional formatting required for custom scalar types
+ * and objects with `__typename` field.
+ *
```
Comment (vyacheslav-pushkin) [inline example-app/src/core/transform/model/serialize.ts:8]: Same

File: packages/codegen/src/generators/react-typescript/app/template/src/app/App.css [hunk 3]
```diff
@@ -64,3 +64,7 @@
 .list-wrapper {
     flex-grow: 1;
 }
+
+form.ant-form .ant-input-number {
+    width: 200px;
+}
```
Comment (vyacheslav-pushkin) [inline packages/codegen/src/generators/react-typescript/app/template/src/app/App.css:70]: Number input is not responsive:

image

**Review by vyacheslav-pushkin — APPROVED** [review]
Please address comments and merge

File: example-app/src/core/transform/model/deserialize.ts (hunk 1 above)
Comment (vadimbasko) [reply example-app/src/core/transform/model/deserialize.ts:8]: fixed in 3rd commit - example app code update

File: packages/codegen/src/generators/react-typescript/app/template/src/app/App.css (hunk 3 above)
Comment (vadimbasko) [reply packages/codegen/src/generators/react-typescript/app/template/src/app/App.css]: fixed

**Pull request merged**